CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+cat /kubelet.env > /etc/kubernetes/kubelet.env
+
+echo -e "\nNew Kubelet env file:\n"
+cat /etc/kubernetes/kubelet.env
+
+echo -e "\nUpdating Kubelet config creator file...\nOld Kubelet config creator file:\n"
+cat /etc/kubernetes/configure-kubelet-cgroup-driver
+
+cp /etc/kubernetes/configure-kubelet-cgroup-driver /configure-kubelet-cgroup-driver
+sed -i "s|^EOF|providerID: external|g" /configure-kubelet-cgroup-driver
+echo "EOF" >> /configure-kubelet-cgroup-driver
+cat /configure-kubelet-cgroup-driver > /etc/kubernetes/configure-kubelet-cgroup-driver
+
+echo -e "\nNew Kubelet config creator file:\n"
+cat /etc/kubernetes/configure-kubelet-cgroup-driver
+
+echo -e "\nRestarting Kubelet...\n"
+exec nsenter -a -t 1 /bin/sh -c "systemctl restart kubelet && systemctl status --no-pager kubelet"
+' | tee /tmp/run
+```
+
+Store above script in a configmap in a separate namespace:
+
+```bash
+kubectl create ns update-host-files
+kubectl create -n update-host-files cm script --from-file /tmp/run --dry-run=client -o yaml | kubectl apply -f -
+```
+
+Run this command to create pods on each node that is running metallb:
+
+```bash
+for nodename in $(kubectl get nodes -l metallb.lokomotive.io/my-asn=65000,lokomotive.alpha.kinvolk.io/bgp-enabled!=true -ojsonpath='{.items[*].metadata.name}')
+# for nodename in $(kubectl get nodes -l metallb.lokomotive.io/my-asn=65000 -ojsonpath='{.items[*].metadata.name}')
+do
+  echo "
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: update-host-files-$nodename
+  name: update-host-files-$nodename
+  namespace: update-host-files
+spec:
+  containers:
+  - image: registry.fedoraproject.org/fedora:32
+    name: update-host-files
+    imagePullPolicy: IfNotPresent
+    securityContext:
+      privileged: true
+    args:
+    - bash
+    - /tmp/script/run
+    volumeMounts:
+    - name: kubelet-env
+      mountPath: /etc/kubernetes/kubelet.env
+    - name: kubelet-config
+      mountPath: /etc/kubernetes/configure-kubelet-cgroup-driver
+    - name: script
+      mountPath: /tmp/script/
+  nodeName: $nodename
+  restartPolicy: Never
+  hostPID: true
+  volumes:
+  - name: kubelet-env
+    hostPath:
+      path: /etc/kubernetes/kubelet.env
+  - name: kubelet-config
+    hostPath:
+      path: /etc/kubernetes/configure-kubelet-cgroup-driver
+  - name: script
+    configMap:
+      name: script
+" | kubectl apply -f -
+
+  while true
+  do
+    complete=$(kubectl -n update-host-files get pod update-host-files-$nodename -o jsonpath='{.status.containerStatuses[-1].state.terminated.exitCode}')
+    #
+    if [ -z "${complete}" ]; then
+      echo "waiting for pod update-host-files-$nodename to finish"
+    else
+      kubectl -n update-host-files logs update-host-files-$nodename
+      break
+    fi
+    sleep 1
+  done
+
+  kubectl label node $nodename lokomotive.alpha.kinvolk.io/bgp-enabled=true
+done
+```
+
+4. Reload the new config for self-hosted kubelet
+
+  If you are running self-hosted kubelet then restart it:
```

Comment:
How do I know if I'm running a self-hosted kubelet? We should either explain or better yet - make the command figure this out on the user's behalf and run conditionally.